sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1600,6 +1600,21 @@ __SYCL_EXPORT pi_result piextUSMGetMemAllocInfo(
     pi_context context, const void *ptr, pi_mem_info param_name,
     size_t param_value_size, void *param_value, size_t *param_value_size_ret);
 
+/// This structure denotes a plugin-specific data returned via the
+/// piextPluginGetOpaqueData PI call. Depending on the \c version field, the second \c data
+/// field is can be interpreted differently. Versioning and data interpretation is plugin-specific.
+struct PluginOpaqueData {
+  uintptr_t version;
+  void *data;
+};
+/// API to get Plugin internal data, opaque to SYCL RT. Some devices whose device code is compiled
+/// by the host compiler (e.g. CPU emulators) may use it to access some device code functionality
+/// implemented in/behind the plugin.
+/// \param opaque_data_param - unspecified argument, interpretation is specific to a plugin
+/// \param opaque_data_return - placeholder for the returned opaque data.
+__SYCL_EXPORT pi_result piextPluginGetOpaqueData(void *opaque_data_param,
+                                                 PluginOpaqueData **opaque_data_return);
+
```

Comment:
Since we agreed not to maintain versioning of opaque data on PI level, this needs to be adjusted once again - the opaque data struct moved back to ESIMD EMU header:
```suggestion
/// API to get Plugin internal data, opaque to SYCL RT. Some devices whose device code is compiled
/// by the host compiler (e.g. CPU emulators) may use it to access some device code functionality
/// implemented in/behind the plugin.
/// \param opaque_data_param - unspecified argument, interpretation is specific to a plugin
/// \param opaque_data_return - placeholder for the returned opaque data.
__SYCL_EXPORT pi_result piextPluginGetOpaqueData(void *opaque_data_param,
                                                 void **opaque_data_return);

```